config/locales/views/repositories/nl.yml

```
@@ -20,6 +20,11 @@ nl:
       exercises: Oefeningen
       content_pages: Leesactiviteiten
       learning_activities: Leeractiviteiten
+      public_files: Publieke bestanden
+      public_files_info: "Bestanden in de /public/ map van de repository worden hier opgelijst en kunnen gebruikt worden in cursus-, reeks- en oefeningenbeschrijvingen onder de url %{url}."
```

```suggestion
      public_files_info: "Bestanden in de <code>/public</code> map van een repository kunnen overal op Dodona gebruikt worden, bijvoorbeeld in cursus-, reeks- en oefeningenbeschrijvingen, via onderstaande url."
```
You will probably need to add `_html` to the key to make the code tag work + add similar change to the English version.